source/built-ins.d.ts

```diff
@@ -0,0 +1,7 @@
+import {Primitive} from './primitive';
+/**
+Matches any primitive, date, or regexp value
+
+@category Type
+*/
+export type BuiltIns = Primitive | Date | RegExp;
```

Comment:
Move it to https://github.com/sindresorhus/type-fest/blob/main/source/internal.d.ts

Reply:
sure thing